.circleci/config.yml

```
@@ -117,6 +117,21 @@ jobs:
           path: /tmp/circular_deps.svg
           destination: /hpx/artifacts/circular_deps.svg
 
+  # Check if the files of the module are listed in the CMakeLists.txt
+  check_cmakelists_files:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /hpx
+      - run:
+          name: Check if the headers of the modules are listed in their CMakeLists.txt
+          command: |
+              ./tools/check_cmakelists_files.sh > /tmp/missing_files.txt
```

This should be `/hpx/source/tools/...` instead, I believe:
```suggestion
              /hpx/source/tools/check_cmakelists_files.sh > /tmp/missing_files.txt
```

---

Oh yeah sorry I was testing it locallly, put a source_dir variable to change it and forgot to use it ^^ thanks !